src/main/java/com/google/jenkins/plugins/k8sengine/KubectlWrapper.java

```diff
@@ -108,4 +144,40 @@ private static String launchAndJoinCommand(Launcher launcher, List<String> args)
 
     return cmdLogStream.toString(CHARSET);
   }
+
+  /**
+   * Using the kubectl cli tool as the api client for the caller, this method unmarshalls the json
+   * output of the cli to a JSON Object.
+   *
+   * @param kind - the kind of kubernetes Object
```

Comment:
Missing trailing "." for \@param and \@return annotations. Use of consistent grammar throughout is preferred.